schema/deploy/shipping/views.sql

```diff
@@ -48,17 +48,19 @@ create or replace view shipping.reportable_condition_v1 as
     order by encountered desc;
 
 
+drop view shipping.metadata_for_augur_build_v2;
 create or replace view shipping.metadata_for_augur_build_v2 as
 
-    select  sample as strain,
-            encountered as date,
+    select sample as strain,
+            cast(encountered as date) as date,
```

Comment:
@tsibley should I also changed to using `encounter_date` instead of casting `encountered` like you did in v3? It would require joining on a different version of `shipping.incidence_model_observation*`, though.

Reply:
I used `encounter_date` in `metadata_for_augur_build_v3` to a) get [correct timezone handling](https://github.com/seattleflu/id3c-customizations/blob/c8d3f7abb889d330b243c2529c017c2eadd9d955/schema/deploy/shipping/views.sql#L267) and b) remove the need for the join `warehouse.encounter`.

For `metadata_for_augur_build_v2` here, I wouldn't change the join, but use the same timezone conversion logic. It may also be that we want to set the default database timezone to US/Pacific.